python/python/glide/async_commands/transaction.py

```diff
@@ -3304,6 +3304,38 @@ def flushall(
             args.append(flush_mode.value)
         return self.append_command(RequestType.FlushAll, args)
 
+    def copy(
```

Comment:
Similar to how you've split copy into two different functions for non-transactions (one for standalone and one for cluster), I think we will have to split this. Instead of putting it under BaseTransaction, we'll have to put one function under Transaction and one under ClusterTransaction (both classes are in this same file, lower down)